setup.py

```diff
@@ -32,14 +32,20 @@
 
     provides=[],
     install_requires=[
-        'beautifulsoup4',
-        'beautifultable',
-        'click',
-        'colorama',
-        'pandas',
-        'requests',
-        'responses',
-        'validators',
+        'beautifulsoup4==4.6.0',
+        'beautifultable==0.5.0',
+        'click==6.7',
+        'coverage==4.5.1',
+        'coveralls==1.3.0',
```

Comment:
Why do we need `coverage` and `coveralls` in requirements? Any specific reason?

Reply:
Sorry, I copied this from requirements. I'll change it.